src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+/**
+ * Parser to parse user input.
+ */
+public class Parser {
+    /**
+     * Parse the user input command and response it correspondingly.
+     *
+     * @param input user input command
+     * @return corresponding command given a specific user input.
+     * @throws DukeUnknownCommandException indicate that an unknown command has been passed.
+     * @throws DukeEmptyArgumentException indicate that a command has been passed an empty argument.
+     * @throws DukeInvalidArgumentException indicate that a command has been passed an illegal argument.
+     */
+    public static Command parse(String input) throws DukeUnknownCommandException,
+            DukeEmptyArgumentException, DukeInvalidArgumentException {
+        String[] split = input.split(" ", 2);
```

Comment:
Might want to consider case insensitive string comparisons

Reply:
fixed!